.github/workflows/sync-main-to-prod.yml

```diff
@@ -77,20 +100,21 @@ jobs:
                 check_expiration repository/repository/$f
             fi;
 
-            gcloud --quiet storage cp --cache-control=no-store -r repository/repository/$f gs://sigstore-preprod-tuf-root/
+            gcloud --quiet storage cp --cache-control=no-store -r repository/repository/$f gs://sigstore-tuf-root/
```

Comment:
Per @kommendorkapten's suggestion in https://github.com/sigstore/root-signing/issues/893#issuecomment-1677180668, can we keep syncing to preprod and also sync to prod? This'll keep the probers happy

Reply:
Updated